.gitignore

```
@@ -0,0 +1,3 @@
+node_modules/
+.vscode/
+package-lock.json
```

No subir la carpeta de build
```suggestion
package-lock.json
build/
```